games-simulation/simutrans/simutrans-120.2.1.recipe

```diff
@@ -0,0 +1,86 @@
+SUMMARY="A transport simulation game"
+DESCRIPTION="Simutrans is a freeware and open-source transportation \
+simulator. Your goal is to establish a successful transport company. \
+Transport passengers, mail and goods by rail, road, ship, and even air. \
+Interconnect districts, cities, public buildings, industries and \
+tourist attractions by building a transport network you always dreamed \
+of."
+HOMEPAGE="http://www.simutrans.com"
+SvnRevision="8121"
+GitRevision="83664a991f819677515ec6bb14e2cf5ce126d8ba"
+SOURCE_URI="https://github.com/aburch/simutrans/archive/$GitRevision.tar.gz"
+CHECKSUM_SHA256="4fb73683e648ddc09964a7e4d58f7928b94a9d28088c68635ae9a20789b39c43"
+SOURCE_FILENAME="simutrans-$portVersion.tar.gz"
+SOURCE_DIR="simutrans-$GitRevision"
+SOURCE_URI_2="https://downloads.sourceforge.net/project/simutrans/simutrans/120-2-1/simutrans-src-120-2-1.zip"
+CHECKSUM_SHA256_2="7dfbd1c44f47ddd8caedb3da77dac8262652f96e1c9ffd890b67757e08ddc942"
+SOURCE_FILENAME_2="simutrans-sf-$portVersion.zip"
+SOURCE_DIR_2=""
+COPYRIGHT="1997-2004 Hj. Malthaner
+	2005-2017 The Simutrans Team"
+LICENSE="Artistic"
+REVISION="1"
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+PROVIDES="
+	simutrans$secondaryArchSuffix = $portVersion
+	app:simutrans$secondaryArchSuffix = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libsdl$secondaryArchSuffix
+	lib:libsdl_mixer$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	lib:libpng$secondaryArchSuffix
+	lib:libbz2$secondaryArchSuffix
+	simutrans_pakset$secondaryArchSuffix >= 120.2
+	timgmsoundfont
+	"
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libsdl$secondaryArchSuffix
+	devel:libsdl_mixer$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	devel:libpng$secondaryArchSuffix
+	devel:libbz2$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:g++$secondaryArchSuffix
+	cmd:gcc$secondaryArchSuffix
+	cmd:autoreconf
+	cmd:aclocal
+	cmd:make
+	cmd:ld$secondaryArchSuffix
+	cmd:strip$secondaryArchSuffix
+	cmd:sdl_config$secondaryArchSuffix
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+GLOBAL_WRITABLE_FILES="
+	non-packaged/$relativeAppsDir/simutrans/config directory keep-old
+	"
+
+BUILD()
+{
+	autoreconf configure.ac
+	export CFLAGS+=-DREVISION=$SvnRevision
+	runConfigure ./configure
+	make $jobArgs
+	strip sim
+}
+
+INSTALL()
+{
+	fixPkgconfig
+	cp $sourceDir2/simutrans/text/*.tab simutrans/text/
+	cp sim simutrans/simutrans
+	mkdir -p $appsDir/simutrans
+	mkdir -p $prefix/non-packaged/$relativeAppsDir/simutrans/config
```

Comment:
Not sure what you are doing here, it is not possible to put things in non-packaged from a package so an rm would do the same?
Also, what about using $settingsDir instead? Is there a reason to not put things there?